include/rocksdb/db.h

```diff
@@ -621,6 +621,10 @@ class DB {
     // "rocksdb.block-cache-pinned-usage" - returns the memory size for the
     //      entries being pinned.
     static const std::string kBlockCachePinnedUsage;
+
+    // "rocksdb.immutable_db_options_statistic" - returns the statistic
+    //      of immutable db options
+    static const std::string kImmutableDBOptionStatistic;
```

Comment:
This name is not clear enough for people who operates this system. They may not know what immutable_db_options is. immutable_db_options is in fact an internal data structure. Telling people the location of the stat object is not helpful. 

We usually call this object `statistics`. Maybe we can just call it `rocksdb.statistics`. I know it's a little bit confusing becasue we already have "rocksdb.stats" and "rocksdb.dbstats". If you think it's too confusing, something like `rocksdb.statistics.in.options` will be better than the name it is currently picked.